product_docs/docs/transporter/2_preview/terminology.mdx

```diff
@@ -0,0 +1,20 @@
+---
+title: "Terminology"
+---
+The terminology that follows is important for understanding EDB Transporter functionality.
+
+## Kafka
+Apache Kafka is an open-source, distributed-event, streaming platform used by thousands of companies for high-performance data pipelines, streaming analytics, data integration, and mission-critical applications.
```

Comment:
Wasn't sure about proper hyphen use. Is it a streaming platform that handles distributed events? Or is it a distributed event-streaming platform?

Reply:
It's a distributed event-streaming platform